README.adoc

```diff
@@ -0,0 +1,478 @@
+
+. First, make sure you're in the *main* worktree.
+. Switch to the required JDK using SDKMAN! by running the following command:
++
+--
+ $ sdk env || sdk env install
+
+SDKMAN! will switch to the required JDK, provisioning it first if it isn't already available on your machine.
+--
+
+. Generate the documentation with Antora using the following command:
++
+--
+ $ ./gradlew -PbuildSrc.skipTests=true :spring-security-docs:antora
+
+You can also run this command directly from the _docs_ folder:
+
+ $ cd docs
+ $ ../gradlew -PbuildSrc.skipTests=true antora
+
+This command will build the documentation for the *main* branch.
+The Antora playbook is retrieved by the playbook provider from the *docs-build* branch.
+The retrieved playbook file will be cached as _cached-antora-playbook.yml_.
+--
+
+. Navigate to the local file URI shown in the terminal to view the generated documentation.
+
+=== Build the documentation in the 6.0.x branch (writers)
+
+NOTE: The instructions in this section assume you've completed the steps in <<prepare-workspace>>.
+
+. First, change to the *6.0.x* worktree.
+
+ $ cd ../6.0.x
+
+. Switch to the required JDK using SDKMAN! by running the following command:
++
+--
+ $ sdk env || sdk env install
+
+SDKMAN! will switch to the required JDK, provisioning it first if it isn't already available on your machine.
+--
+
+. Generate the documentation with the following command:
++
+--
+ $ ./gradlew -PbuildSrc.skipTests=true :spring-security-docs:antora
+
+This command will build the documentation for the *6.0.x* branch.
+The Antora playbook is retrieved by the playbook provider from the *docs-build* branch.
+The retrieved playbook file will be cached as _cached-antora-playbook.yml_.
+--
+
+. Navigate to the local file URI shown in the terminal to view the generated documentation.
+
+=== Build the documentation for production (docs developer)
+
+NOTE: The instructions in this section assume you've completed the steps in <<prepare-workspace>>.
+
+To build the project's production site, you'll set up a worktree for the *docs-build* branch of the repository.
+
+. To add a worktree, you have to be in the main worktree.
+In your terminal, change to the *main* worktree if you aren't already in it.
+
+ $ cd ../main
+
+. Run the following command to set up the worktree for the *docs-build* branch.
+Then change into the new _docs-build_ directory.
+
+ $ git worktree add ../docs-build docs-build --track
+ $ cd ../docs-build
+
+. Switch to the required JDK or install it.
+
+ $ sdk env || sdk env install
+
+. Generate the documentation for the project's production site using the following command:
++
+--
+ $ ./gradlew antora
+
+This command will build all of the documentation for the production site from the git repository on GitHub.
+
+(Optional) To build the documentation using the current clone, using any available worktrees, run the following command instead:
+
+ $ ./gradlew antora --playbook local-antora-playbook.yml
```

Comment:
I see no `local-antora-playbook.yml` file added in this PR, nor in #4422. Is that a typo or the file is missing from the change set?